common_logging/src/test/java/uk/gov/dhsc/htbhf/logging/AbstractLoggingTest.java

```diff
@@ -22,4 +22,12 @@ void assertCorrectLoggingMessage(String expectedMessage, Level expectedLevel) {
         assertThat(loggingEvent.getMessage()).isEqualTo(expectedMessage);
         assertThat(loggingEvent.getLevel().toString()).isEqualTo(expectedLevel.toString());
     }
+
+    String getSingleMessageContent(Level expectedLevel) {
```

Comment:
Can we combine this with the code above, there is a lot of commonality?

Reply:
refactored into `assertSingleLogMessageContainsText`